events/import_user_events_big_query.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START retail_import_user_events_from_big_query]
+# Import user events into a catalog from BigQuery using Retail API
+#
+import os
+import time
+
+from google.api_core.client_options import ClientOptions
+from google.cloud.retail import BigQuerySource, UserEventInputConfig, UserEventServiceClient, \
+    ImportUserEventsRequest
+
+project_number = os.getenv('PROJECT_NUMBER')
+
+# TODO(developer) Define the project Id here:
+project_id = ""
+
+endpoint = "retail.googleapis.com"
+default_catalog = "projects/{0}/locations/global/catalogs/default_catalog".format(project_number)
+dataset_id = "user_events"
+#table_id = "import_tutorial"
```

Comment:
it would be better to have valid table un-commented for first execution

Reply:
done